src/components/CurriculumComponents/CurricSEOAccordion/CurricSEOAccordion.test.tsx

```diff
@@ -0,0 +1,109 @@
+import { render, screen, fireEvent } from "@testing-library/react";
+import { PortableTextBlock } from "@portabletext/react";
+
+import { CurricSEOAccordion } from "./CurricSEOAccordion";
+
+import { mockPortableTextBlocks } from "@/components/CurriculumComponents/CurriculumVisualiser/fixtures";
```

Comment:
Can we move this fixture over into this directory. Best practice to not use things from other components.